backend/requirements.txt

```diff
@@ -94,3 +94,5 @@ urllib3==2.3.0
     # via
     #   requests
     #   types-requests
+pyclamd

```

Comment:
Not understanding why this is duplicated here.

Reply:
Just checking for clarity: Won't we need the file scan for production, so it needs to be in both requirement files?